app/index.html

```diff
@@ -9,10 +9,10 @@
   <title>My AngularJS App</title>
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="bower_components/html5-boilerplate/dist/css/normalize.css">
-  <link rel="stylesheet" href="bower_components/html5-boilerplate/dist/css/main.css">
+  <link rel="stylesheet" href="lib/html5-boilerplate/dist/css/normalize.css">
```

Comment:
Because the root directory (for the webapp) is `app/` which is a sibling of `node_module/`.